### DISCUSSION TITLE
customize the home page with html / css

### DISCUSSION THREAD

**Discussion by leonardo75018** [requarks/wiki#3193] [Help / Questions] [answered]
Hello the community.
I would like to know if it is possible to customize the style of the home page in the CSS/html file.

If possible can you please tell me how I can find the html/css file of the home page! 

Thank you very much

**Comment by NGPixel** [comment, marked as answer]
CSS can be modified via the Admin > Theme page.

As for HTML, you would usually edit the /home page directly, like any other page.
